etc/chromeos/lib/installs.sh

```diff
@@ -106,3 +106,14 @@ install_nomad_chromeos_workaround() {
     sudo mkdir -p "/lib/modules/$(uname -r)/"
     echo '_/bridge.ko' | sudo tee -a "/lib/modules/$(uname -r)/modules.builtin"
 }
+
+nuke_home_venv() {
+    # Apparently if you have ~/.venv that breaks the pyenv install, so we nuke it.
```

Comment:
Oops, I mis-reported this. I had to nuke my `~/.pyenv` directory >_<. Sorry.

Reply:
Ah, that makes more sense. I'll fix now